pilot/pkg/model/push_context.go

```diff
@@ -390,6 +390,9 @@ type PushRequest struct {
 	// Delta defines the resources that were added or removed as part of this push request.
 	// This is set only on requests from the client which change the set of resources they (un)subscribe from.
 	Delta ResourceDelta
+
+	// Forced defines that configs should be generated and pushed regardless if they have changed or not.
+	Forced bool
```

Comment:
One thing I am worried about is now its very easy for someone to accidentally look at ConfigsUpdate (which will be updated and populated) while accidentally ignoring Forced.

Forced means "everything [might have] changed" so that isn't valid.

Reply:
Isn't the same thing possible today? I mean, one could only look that ConfigsUpdated is empty and think that nothing has changed 🤔

Reply:
@howardjohn I also think that being so explicit exposes stuff that we probably didn't really know about or hadn't realized, for example, in Delta, when we process a request from the proxy that is deemed to need a response, it's always triggering a Forced push, I'm not really sure, but it may not be what we always want.